tapdance/getifaddr_test.go

```diff
@@ -0,0 +1,13 @@
+package tapdance
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestIpV6Support(t *testing.T) {
+	fmt.Println("Testing IPv6 Support")
+
+	fmt.Printf("Supports IPV6: %t\n", SupportsIpv6())
```

Comment:
This isn't going to work consistently because SupportsIPv6 checks the real interfaces on the device that is running the test. This may mean that it is worth removing this test, but the result of this test will depend on whether you have a legitimate ipv6 address.

Reply:
tests updated